Feature/timi/android getting started (#593)
MapsIndoors/docs: src/android/v3/getting-started/new-project.md

<!-- Environment -->
{% include "src/shared/getting-started/project/environment.md" %}

This guide explains how to start using a MapsIndoors map in your Android application using the MapsIndoors Android SDK v3.

To benefit from the guides, you will need basic knowledge about:

* Android Development
* Google Maps Android API

You can get started in two ways, either by reviewing and modifying the [basic example](#basic-example) or do the [clean setup](#setup-mapsindoors).

## Basic Example

You will find in the **Create a link to a very basic implementation of a single activity mapsindoors implementation** a repo that contains the minimum code to start a MapsIndoors project.

> You can also follow the steps below to start your app from scratch or to enhance the Basic Examples, more features will be explained in the [guides](/android/v3/guides/).

Reviewer: ```suggestion

You can also follow the steps below to start your app from scratch or to enhance the Basic Examples, more features will be explained in the [guides](/android/v3/guides/).
```

## Setup MapsIndoors

> If you dont already have a project. We recommend using the Google Maps Activity preset from android studio to getting started on developing your MapsIndoors solution.

Reviewer: ```suggestion
If you don't already have a project, we recommend using the Google Maps Activity preset from Android Studio to getting started on developing your MapsIndoors project.
```

> Add the MapsIndoors SDK as a dependency to your project. The AAR for the MapsIndoors SDK contains both Java classes, SDK resources and AndroidManifest.xml template which gets merged into your application's AndroidManifest.xml during build process. Add or merge in the following to your app's build gradle file (usually called build.gradle).

Reviewer: ```suggestion
Add the MapsIndoors SDK as a dependency to your project. The _AAR_ for the MapsIndoors SDK contains both Java classes, SDK resources and an `AndroidManifest.xml` template which gets merged into your application's `AndroidManifest.xml` during build process. Add or merge in the following to your app's build gradle file (usually called `build.gradle`).
```

Make sure that the minimum Android SDK version is 21 (aka. "Android Lollipop", version 5.0) or above:

```java
android {
    defaultConfig {
        minSdkVersion 21
    }
    ...
}
```

> MapsIndoors rely on Java 8 features, so you must add the following compile options, also in *android* section of your *build.gradle* file:

Reviewer: ```suggestion
MapsIndoors relies on Java 8 features, so you must add the following compile options, also in *android* section of your *build.gradle* file:
```

<!-- Environment -->
{% include "src/shared/getting-started/project/environment.md" %}

This guide explains how to start using a MapsIndoors map in your Android application using the MapsIndoors Android SDK v3.

To benefit from the guides, you will need basic knowledge about:

* Android Development
* Google Maps Android API

You can get started in two ways, either by reviewing and modifying the [basic example](#basic-example) or do the [clean setup](#setup-mapsindoors).

## Basic Example

You will find in the **Create a link to a very basic implementation of a single activity mapsindoors implementation** a repo that contains the minimum code to start a MapsIndoors project.

> You can also follow the steps below to start your app from scratch or to enhance the Basic Examples, more features will be explained in the [guides](/android/v3/guides/).

Reviewer: ```suggestion
You can also follow the steps below to start your app from scratch or to enhance the Basic Examples, more features will be explained in the [guides]({{ site.url }}/android/v3/guides/).
```

## Setup MapsIndoors

If you don't already have a project, we recommend using the Google Maps Activity preset from Android Studio to getting started on developing your MapsIndoors project.

Add the MapsIndoors SDK as a dependency to your project. The _AAR_ for the MapsIndoors SDK contains both Java classes, SDK resources and an `AndroidManifest.xml` template which gets merged into your application's `AndroidManifest.xml` during build process. Add or merge in the following to your app's build gradle file (usually called `build.gradle`).

Make sure that the minimum Android SDK version is 21 (aka. "Android Lollipop", version 5.0) or above:

```java
android {
    defaultConfig {
        minSdkVersion 21
    }
    ...
}
```

> MapsIndoors relies on Java 8 features, so you must add the following compile options, also in *android* section of your *build.gradle* file:

Author: Yes, i am pretty sure it is. Our SDK uses a lot of java 8 features.

> MapsIndoors relies on Java 8 features, so you must add the following compile options, also in *android* section of your *build.gradle* file:

Reviewer 2: Ok, I was just asking because I seem to recall it has been like this for quite some time...

> MapsIndoors relies on Java 8 features, so you must add the following compile options, also in *android* section of your *build.gradle* file:

Author: Yearh, a guess would be that most modern applications use it anyway. It's features like lambda's that are removed if you dont use java 8 😬  
You can read about it here if you are interested: https://developer.android.com/studio/write/java8-support
